### PR TITLE
Added new WebPageContentProvider for links to html pages.

### PR DESCRIPTION
**Pull request by jasonpbarrera**
Gives simple title, description, etc..

### CODE REVIEW

File: Chat/ContentProviders/WebPageContentProvider.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            public  string Email { get; set; }
+            public  string PhoneNumber { get; set; }
+            public  string FaxNumber { get; set; }
+            public  string Image { get; set; }
+
+            public override string  ToString() {
+ 	            StringBuilder sb = new StringBuilder();
+
+                sb.Append("<div class=\"webPageContent\">");
+                sb.AppendFormat("<div><a href=\"{0}\" class=\"webPageLink\">{1}</a></div>", Url, Title);
```
Comment (davidfowl) [inline Chat/ContentProviders/WebPageContentProvider.cs:60]: Seems redundant to put the link again.

File: Chat/ContentProviders/WebPageContentProvider.cs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+using System.Linq;
+using System.Web;
+using SignalR.Samples.Hubs.Chat;
+using HtmlAgilityPack;
+using System.Net;
+using System.IO;
+using System.Text;
+
+// Credit
+// http://webcmd.wordpress.com/2011/02/16/c-webservice-for-getting-link-details-like-facebook/
```
Comment (davidfowl) [inline Chat/ContentProviders/WebPageContentProvider.cs:13]: What about the images? That's the best part of it...

Comment (jasonpbarrera) [inline Chat/ContentProviders/WebPageContentProvider.cs:13]: I didn't put that in - Not sure about which image to grab. Unless it's using open graph.

File: Chat/ContentProviders/WebPageContentProvider.cs (hunk 1 above)
Comment (jasonpbarrera) [inline Chat/ContentProviders/WebPageContentProvider.cs:60]: Yes, in away. I wanted to reformat the url though to make it look nicer. The code, I think, needs to be changed to allow processing before outputting the first link. Facebook and others pretty much do the same thing.

File: Chat/ContentProviders/WebPageContentProvider.cs (hunk 2 above)
Comment (davidfowl) [inline Chat/ContentProviders/WebPageContentProvider.cs:13]: The article does a guess, seems fine to me.

Comment (jasonpbarrera) [inline Chat/ContentProviders/WebPageContentProvider.cs:13]: OK, just didn't want any pronz leaking in. :)

Comment (davidfowl) [inline Chat/ContentProviders/WebPageContentProvider.cs:13]: hahahahah true. That does kinda suck :).

File: Chat/Chat.js [hunk 3]
```diff
@@ -139,7 +139,8 @@ $(function () {
         }
         else {
             $.each(rooms, function () {
-                addMessage(this.Name + ' (' + this.Count + ')');
+                var msg = addMessage('<a href="#" class="joinroomlink">' + this.Name + '</a>' + ' (' + this.Count + ')');
```
Comment (davidfowl) [inline Chat/Chat.js:142]: Nice feature but this needs a separate pull request.

Comment (jasonpbarrera) [inline Chat/Chat.js:142]: Can't do any more pull requests. I guess delete it!

**Pull request closed without merging**